lib/viral_spiral/room/card_draw.ex

```diff
@@ -102,6 +102,10 @@ defmodule ViralSpiral.Room.CardDraw do
         ^identity_count ->
```

Comment:
@maanasb01 it took me a few re-reads to realize that what you are trying to handle here is the case when the number of players is equal to the number of total identities supported by viral spiral(3 here)

I wonder if the code was more readable if we created 2 conditions within this case.
one when `total` is 2 or 3, in which case just return `Enum.shuffle(room_communities)` and for all other cases (`_`) do whats already written.

Lmk if I am missing something.

Reply:
Haan sure that works. My main issue was that it was hard to understand what scenario `^identity_count` was handling. With what you are suggesting, it would be clear that there is a default scenario and some special scenario for when the player count is 2.